Sources/Components/Profile/IdentityView.swift

```diff
@@ -0,0 +1,152 @@
+
+        if let description = viewModel.description?.trimmingCharacters(in: .whitespacesAndNewlines), !description.isEmpty {
+            label.text = description
+        } else {
+            label.isHidden = true
+        }
+
+        return label
+    }()
+
+    // MARK: - Setup
+
+    public required init?(coder aDecoder: NSCoder) {
+        fatalError("\(#function) not implemented")
+    }
+
+    public required init(viewModel: IdentityViewModel) {
+        self.viewModel = viewModel
+        super.init(frame: .zero)
+        setupSubviews()
+        addTapListener()
+    }
+
+    private func setupSubviews() {
+        layer.cornerRadius = 8
+        backgroundColor = .ice
+
+        addSubview(profileImageView)
+        addSubview(profileNameLabel)
+        addSubview(verifiedBadge)
+        addSubview(subtitleLabel)
+
+        NSLayoutConstraint.activate([
+            profileImageView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: .mediumLargeSpacing),
+            profileImageView.topAnchor.constraint(equalTo: topAnchor, constant: .mediumLargeSpacing),
+            profileImageView.bottomAnchor.constraint(lessThanOrEqualTo: bottomAnchor, constant: -.mediumLargeSpacing),
+            profileImageView.widthAnchor.constraint(equalToConstant: profileImageSize),
+            profileImageView.heightAnchor.constraint(equalToConstant: profileImageSize),
+
+            profileNameLabel.leadingAnchor.constraint(equalTo: profileImageView.trailingAnchor, constant: .mediumSpacing),
+            profileNameLabel.topAnchor.constraint(equalTo: profileImageView.topAnchor),
+
+            verifiedBadge.leadingAnchor.constraint(equalTo: profileNameLabel.trailingAnchor, constant: .smallSpacing),
+            verifiedBadge.centerYAnchor.constraint(equalTo: profileNameLabel.centerYAnchor),
+            verifiedBadge.trailingAnchor.constraint(lessThanOrEqualTo: trailingAnchor, constant: -.mediumSpacing),
+            verifiedBadge.widthAnchor.constraint(equalToConstant: 18),
+            verifiedBadge.heightAnchor.constraint(equalToConstant: 18),
+
+            subtitleLabel.leadingAnchor.constraint(equalTo: profileImageView.trailingAnchor, constant: .mediumSpacing),
+            subtitleLabel.topAnchor.constraint(equalTo: profileNameLabel.bottomAnchor, constant: .verySmallSpacing),
+            subtitleLabel.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -.mediumLargeSpacing),
+        ])
+
+        if !descriptionLabel.isHidden {
+            addSubview(descriptionLabel)
+            NSLayoutConstraint.activate([
+                descriptionLabel.leadingAnchor.constraint(equalTo: leadingAnchor, constant: .mediumLargeSpacing),
+                descriptionLabel.topAnchor.constraint(greaterThanOrEqualTo: profileImageView.bottomAnchor, constant: .mediumLargeSpacing),
+                descriptionLabel.topAnchor.constraint(greaterThanOrEqualTo: subtitleLabel.bottomAnchor, constant: .mediumLargeSpacing),
```

Comment:
These two constraints exists to make sure that we have the correct spacing between the `subtitleLabel` and `descriptionLabel` in case the device is using a larger than default font-size, as the `maxY` of the `subtitleLabel` would then exceed that of `profileImageView`.

If it's too cryptic, I'll add a comment about this 🙂